packages/core/profiler/package.json

```diff
@@ -20,12 +20,13 @@
     "node": ">= 12.0.0"
   },
   "scripts": {
-    "build-ts": "flow-to-ts src/*.js --write && tsc --emitDeclarationOnly --declaration --esModuleInterop --target es2015 --moduleResolution node16 --module node16 src/*.ts && mkdir -p lib && mv src/*.d.ts lib/. && rm src/*.ts && node build-ts.js",
+    "build-ts": "flow-to-ts src/*.js --write && rm -f ./src/*.d.ts && tsc --emitDeclarationOnly --declaration --esModuleInterop --target es2015 --moduleResolution node16 --module node16 src/*.ts && mkdir -p lib && mv src/*.d.ts lib/. && rm src/*.ts && node build-ts.js",
```

Comment:
What's context on this change?

Reply:
Good question ; let me remove this, I'm not sure

Reply:
Okay the issue here is that `flow-to-ts` generates both `*.d.ts` and `*.ts` files.

Then when we run `tsc --emitDeclarationOnly ...` it fails because the `*.d.ts` files are already there.

This is broken on `v2` (`build-ts` fails on master due to this) & should be fixed by this change